testdata/BUILD

```diff
@@ -31,7 +31,7 @@ exports_files(["Runfiles.java"])
 
 genrule(
```

Comment:
could we instead add a target like this one in https://github.com/bazelbuild/rules_docker/blob/master/tests/docker/BUILD + a target like gen_image (also add to https://github.com/bazelbuild/rules_docker/blob/master/tests/docker/BUILD) and then add a container_image test that verifies the resulting file is present in the container (example config for container_test: https://github.com/bazelbuild/rules_docker/blob/master/tests/docker/configs/test.yaml#L3)

Reply:
Oh that's sweet! Added new-style tests.